pkg/envoy/bootstrap/config_test.go

```diff
@@ -110,6 +110,11 @@ static_resources:
         '@type': type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
         explicit_http_config:
           http2_protocol_options: {}
+    upstream_connection_options:
+      tcp_keepalive:
+        keepalive_interval: 5
+        keepalive_probes: 5
+        keepalive_time: 60
```

Comment:
I believe the `upstream_connection_options` also needs to be added to the `reversedExpectedYAML`.

Reply:
+1 👍🏾